diff -ur active_record-0.5-orig/lib/active_record/connection_adapters/mysql_adapter.rb active_record-0.5/lib/active_record/connection_adapters/mysql_adapter.rb
--- active_record-0.5-orig/lib/active_record/connection_adapters/mysql_adapter.rb	2004-02-22 08:40:36.000000000 -0700
+++ active_record-0.5/lib/active_record/connection_adapters/mysql_adapter.rb	2004-02-22 17:09:49.163335968 -0700
@@ -87,7 +90,11 @@
         
         def string_to_time(string)
           time_array = ParseDate.parsedate(string)
-          Time.mktime(time_array[0], time_array[1], time_array[2], time_array[3], time_array[4], time_array[5])
+			 begin
+				 Time.mktime(time_array[0], time_array[1], time_array[2], time_array[3], time_array[4], time_array[5])
+			 rescue ArgumentError
+				 nil
+			 end
         end
     end
   end
